modules/core/src/v2/wallet.ts

```diff
@@ -483,7 +484,7 @@ export class Wallet {
       'feeRate', 'gasLimit', 'gasPrice', 'idfSignedTimestamp', 'idfUserId', 'idfVersion', 'instant',
       'lastLedgerSequence', 'ledgerSequenceDelta', 'maxFee', 'maxFeeRate', 'maxValue', 'memo', 'message', 'minConfirms',
       'minValue', 'noSplitChange', 'numBlocks', 'recipients', 'reservation', 'sequenceId', 'strategy',
-      'targetWalletUnspents', 'trustlines', 'type', 'unspents', 'validFromBlock', 'validToBlock',
+      'targetWalletUnspents', 'trustlines', 'type', 'unspents', 'validFromBlock', 'validToBlock', 'walletContractAddress',
```

Comment:
Why does it need to even be a whitelisted prebuild param? This can be found on the wallet object at `wallet.coinSpecific.baseAddress`, so I don't see why we would ever need to explicitly pass it

Reply:
You're right